app/ide-desktop/lib/client/src/index.js

```diff
@@ -237,6 +237,12 @@ optParser.options('data-gathering', {
     default: true,
 })
 
+optParser.options('mixpanel-token', {
+    describe: 'Mixpanel token used for gathering the user data.',
+    type: 'string',
+    requiresArg: true,
+})
```

Comment:
We need to put some default here. Otherwise you'd need to run IDE with `enso --mixpanel-token` flag every time, which is not user friendly.